src/coldfront_plugin_api/scim_v2/users.py

```diff
@@ -92,7 +92,12 @@ def post(self, request, format=None):
                 "email": email_list[0].get("value", "")
             }
 
-        user = utils.create_user(**found)
+        user = utils.create_user(
+            username=username,
+            email=found.get("email", ""),
```

Comment:
Should these gets have defaults? Any accounts that don't have these keys should probably be skipped given that there's likely something wrong with them, no?

Reply:
The SCIM specification only defines `userName` as required, hence I decided not to fail on those empty values.

Additionally, the values for first name and last name will be overridden upon authentication to ColdFront from Keycloak (via the authentication plugin), so they don't matter particularly and will be reconciled.

I don't care strongly either way. If you'd prefer a Bad Request error on empty values, I can do that change. Just let me know.

Reply:
Sure, that's fine. Just noting it appears the get_or_fetch_user function in utils.py makes a similar call without defaults in case we want to update that for consistency?

Reply:
@jtriley Updated.